src/tests/functional/shared_test_classes/src/subgraph/memory_LSTMCell.cpp

```diff
@@ -2,332 +2,356 @@
+    weights_vals = ov::test::utils::generate_float_numbers(4 * hiddenSize * inputSize, -0.1f, 0.1f);
+    reccurrenceWeights_vals = ov::test::utils::generate_float_numbers(4 * hiddenSize * hiddenSize, -0.1f, 0.1f);
+    bias_vals = ov::test::utils::generate_float_numbers(4 * hiddenSize, -0.2f, 0.1f);
+
+    ov::ParameterVector input_parameter{std::make_shared<ov::op::v0::Parameter>(element_type, ov::Shape(input_dims))};
+    input_parameter[0]->set_friendly_name("Parameter_1");
+
+    auto input_add_const = ngraph::builder::makeConstant(element_type, input_dims, input_bias);
+    auto add = ngraph::builder::makeEltwise(input_parameter[0], input_add_const, ov::test::utils::EltwiseTypes::ADD);
+
+    auto input_mul_const = ngraph::builder::makeConstant(element_type, input_dims, input_weights);
+    auto mul = ngraph::builder::makeEltwise(add, input_mul_const, ov::test::utils::EltwiseTypes::MULTIPLY);
+
+    auto unsqueeze_input_const = std::make_shared<ov::op::v0::Constant>(element::i64, Shape{1}, squeeze_axes);
+    auto unsqueeze_input = std::make_shared<ov::op::v0::Unsqueeze>(mul, unsqueeze_input_const);
+
+    auto permute_in_params = std::make_shared<ov::op::v0::Constant>(element::i64, Shape{3}, Shape{{1, 0, 2}});
+    auto permute_in = std::make_shared<ov::op::v1::Transpose>(unsqueeze_input, permute_in_params);
+
+    auto cell_memory_constant = ngraph::builder::makeConstant<float>(element_type, cell_memory_dims, cell_memory_init);
+    auto var_cell =
+        std::make_shared<Variable>(VariableInfo{PartialShape(cell_memory_dims), element_type, "cell_state_1"});
+    auto var_hidden =
+        std::make_shared<Variable>(VariableInfo{PartialShape(cell_memory_dims), element_type, "hidden_state_1"});
+    auto cell_memory_read = std::make_shared<ov::op::v6::ReadValue>(cell_memory_constant, var_cell);
+
+    auto hidden_memory_constant =
+        ngraph::builder::makeConstant<float>(element_type, hidden_memory_dims, hidden_memory_init);
+    auto hidden_memory_read = std::make_shared<ov::op::v6::ReadValue>(hidden_memory_constant, var_hidden);
+
+    // Body - inputs
+    auto X = std::make_shared<ov::op::v0::Parameter>(element_type, Shape{1, 1, inputSize});
+    auto H_t = std::make_shared<ov::op::v0::Parameter>(element_type, Shape{1, hiddenSize});
+    auto C_t = std::make_shared<ov::op::v0::Parameter>(element_type, Shape{1, hiddenSize});
+    // Body - layers
+    auto squeeze_const = std::make_shared<ov::op::v0::Constant>(element::i64, Shape{1}, squeeze_axes);
+    auto squeeze = std::make_shared<ov::op::v0::Squeeze>(X, squeeze_const);
+
+    auto weightsNode = ngraph::builder::makeConstant<float>(element_type, {4 * hiddenSize, inputSize}, weights_vals);
+    auto reccurrenceWeightsNode =
+        ngraph::builder::makeConstant<float>(element_type, {4 * hiddenSize, hiddenSize}, reccurrenceWeights_vals);
+    auto biasNode = ngraph::builder::makeConstant<float>(element_type, {4 * hiddenSize}, bias_vals);
```

Comment:
Can be migrated to use some helper or `Constant` from `ov` namespace?

Reply:
@olpipi is working on refactor ngraph builder functions, for example:
https://github.com/openvinotoolkit/openvino/pull/21181
https://github.com/openvinotoolkit/openvino/pull/21133

Maybe we can migrate to new ov::test::utils::make_constant method? @olpipi please comments it, thanks!

Reply:
Hi new ov::test::utils::make_constant is not ready yet. I will update it later.
So please merge it without waiting for it.